src/pointer/ptr.rs

```diff
@@ -539,6 +539,19 @@ mod _transitions {
             unsafe { Ptr::from_ptr(self) }
         }
 
+        /// Recalls that `Ptr`'s referent is validly-aligned for `T`.
+        #[inline]
+        pub(crate) fn recall_aligned(
```

Comment:
To me this name reads like "to recall a product", which made me think it was saying "forget that this is aligned." Maybe `into_aligned`?

Reply:
Still need to address this.

Reply:
Done!